public/js/cita.json

```diff
@@ -5008,5 +4868,305 @@
     "lng": -2.5934541,
     "id": "94e94f50-1d30-49f0-8959-b2a799ba3423",
     "booking_location_id": "b74377da-d06b-48d0-9d74-cc362bbb53b3"
+  },
+  {
+    "title": "Poyton Civic Hall",
+    "address": "Park Lane\nCheshire East\nPoyton\nSK12 1GP",
+    "phone": null,
+    "hours": null,
+    "lat": 53.3494644,
+    "lng": -2.1180696,
+    "id": "dc4e84b0-ab6d-44a3-8090-d633b5ed5313",
+    "booking_location_id": "df38d9b9-e9a9-4bf0-b56b-4309965b815c"
+  },
+  {
+    "title": "Peterlee Macmillan Information Centre",
```

Comment:
Again, I would set the title to "Peterlee" and set "Peterlee Macmillan Information Centre" as the first line of the address.